libs/wire-api/src/Wire/API/Connection.hs

```diff
@@ -68,27 +65,23 @@ data UserConnectionList = UserConnectionList
   }
   deriving stock (Eq, Show, Generic)
   deriving (Arbitrary) via (GenericUniform UserConnectionList)
+  deriving (FromJSON, ToJSON, S.ToSchema) via (P.Schema UserConnectionList)
 
+instance P.ToSchema UserConnectionList where
+  schema =
+    P.object "UserConnectionList" $
+      UserConnectionList
+        <$> clConnections P..= P.field "connections" (P.array P.schema)
+        <*> clHasMore P..= P.fieldWithDocModifier "has_more" (P.description ?~ "Indicator that the server has more connections than returned.") P.schema
+
+-- TODO remove
```

Comment:
actually it can't be removed yet as it's still in use; this can only be removed once more endpoints have been servantified (in future PRs). Removed the comment.